intyg-proxy-service/app/src/main/resources/application.properties

```diff
@@ -8,5 +8,8 @@ integration.hsa.gethealthcareunitmembers.endpoint=${integration.ntjp.baseurl}/in
 integration.hsa.gethealthcareprovider.endpoint=${integration.ntjp.baseurl}/infrastructure/directory/organization/GetHealthCareProvider/1/rivtabp21
 integration.hsa.getlastupdate.endpoint=${integration.ntjp.baseurl}/infrastructure/directory/authorizationmanagement/GetHospLastUpdate/1/rivtabp21
 integration.hsa.getunit.endpoint=${integration.ntjp.baseurl}/infrastructure/directory/organization/GetUnit/4/rivtabp21
+integration.pu.cache.seconds=1000
```

Comment:
Since this value should be different in devtest/prod, maybe we should move it to application-dev so that we are forced to set this value.

Reply:
I saw that u already added this to dev, so just remove this one!

Reply:
Matias specifically asked me to add it here as well as in dev (if I didn't misunderstand). @mww-sopra-steria remove or keep? :)